packages/lib-react-components/src/AnimatedNumber/AnimatedNumber.spec.jsx

```diff
@@ -103,13 +103,10 @@ describe('AnimatedNumber', function () {
     })
   })
 
+  // Two tests are skipped here because they throw an error
+  // Error messages are unclear, but might have to do with window.matchMedia,
+  // which is also mocked in test/setup.js
   describe('with animation', function() {
```

Comment:
Two tests are skipped in AnimatedNumber, and it's not worth the time to deep dive into figuring out why they throw errors with unhelpful error messages. The error message is just `123456` with no indication of the code line.